WordPress/src/main/java/org/wordpress/android/models/ReaderCardRecommendedBlog.kt

```diff
@@ -0,0 +1,12 @@
+package org.wordpress.android.models
+
+import com.google.gson.annotations.SerializedName
+
+data class ReaderCardRecommendedBlog(
+    @SerializedName("name") val name: String,
+    @SerializedName("url") val url: String,
+    @SerializedName("blog_id") val blogId: Long,
+    @SerializedName("description") val description: String,
+    @SerializedName("icon") val iconUrl: String?,
+    @SerializedName("feed_id") val feedId: Long?
```

Comment:
`feedId` is nullable for now but I'm waiting for confirmation that this is how API behaves (and not e.g. setting 0 when not available)

Reply:
Currently, both `feed_id` and `icon` fields would be omitted but it will be changed and they will be null if not available. 
Gson will handle both these cases correctly so the model is correct and the comment can be resolved.